docs/public/content/user/support/faq.md

```diff
@@ -111,6 +111,19 @@ WARN com.sap.psr.vulas.backend.BackendConnector - App [group=com.acme, artifact=
 - If an application does neither have any dependencies nor source code, e.g., in folder `src/main/java`, it will not be uploaded to the backend (depending on configuration parameter `vulas.core.app.uploadEmpty`, which defaults to `false`). In this case, the execution of goal execution information cannot be uploaded. Typical examples of _empty_ applications are Maven aggregator projects (`<packaging>pom</packaging>`).
 - If an application version is analyzed for the first time, the initial execution of the `clean` goal will also result in this warning message.
 
+### Why does @@PROJECT_NAME@@ show vulnerabilities which seems to be related to others components
```

Comment:
Maybe you can add that those findings are highlighted with something like "Rebundles: com.fasterxml.jackson.core : jackson-databind : 2.9.8"

Reply:
I did these comments at first, but then I discussed the point with Serena and there is only 1 situation out of 4 where the tool is providing this extra line (refactoring vulnerable code with construct changes). So I was deleting this part in my comments